Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Select.hs

```diff
@@ -54,22 +57,36 @@ import qualified Storage.Queries.Quote as QQuote
 import qualified Storage.Queries.SearchRequest as QSearchRequest
 import Tools.Error
 
+newtype DSelectReq = DSelectReq
```

Comment:
why this type?

Reply:
> why this type?

Coz user have to pass his extra fee somehow?

Reply:
> why this type?

It was used for old select EP, which is now deprecated. Btw should we create select3 EP? request type changed again

Reply:
we don't need to create select3. select2 was created because `select` was not having any request type. As we already have request type for select2, so we can just add params in that req itself with Maybe

Reply:
old select never had any request type. Maybe you only add it while doing changes for customer tip. We can remove this type I think.